### ISSUE TITLE
GPS303

### ISSUE THREAD

**Issue by manjoel** [tananaev/traccar#1665]
Hallo,

I'm using the gps303 with Protol 103 (port 5001). And no mater what configuration i have on the tracker, for the moment (fix002h***n123456 , distance123456 0400 and angle123456 030) the tracker send an position every 3 Minutes and among i receive ever minute:

2016-01-16 11:51:45 DEBUG: [5D12CACD: 5001 < 194.154.216.90] HEX: 3435393731303034303132313232313b
2016-01-16 11:51:45 DEBUG: [5D12CACD: 5001 > 194.154.216.90] HEX: 4f4e
2016-01-16 11:52:38 DEBUG: [5D12CACD: 5001 < 194.154.216.90] HEX: 3435393731303034303132313232313b
2016-01-16 11:52:38 DEBUG: [5D12CACD: 5001 > 194.154.216.90] HEX: 4f4e
2016-01-16 11:53:47 DEBUG: [5D12CACD: 5001 < 194.154.216.90] HEX: 3435393731303034303132313232313b
2016-01-16 11:53:47 DEBUG: [5D12CACD: 5001 > 194.154.216.90] HEX: 4f4e

**Comment by manjoel** [comment]
the messages every 3 Munites are ACC alarms but i'm sure having deactivated acc alarm and the check command says acc: off

**Comment by tananaev** [comment]
I guess it's better to contact seller or manufacturer of the device. I don't think anything can be done on the server side.

**Comment by nelim** [comment]
did you powered the acc wire of the tracker properly?

**Comment by manjoel** [comment]
No for the moment it is running on the internal battery

**Comment by nelim** [comment]
you can try to power the acc wire and see if it will start to upload gps data, maybe this is device bug

**Comment by manjoel** [comment]
12v together with the power wire?

**Comment by kuldiplive** [comment]
Try this :+1: 

1)	begin123456  reset device config
2)	apn123456 yourapn
       3)    adminip123456<1SPACE>1.1.1.1<1SPACE>1111     your ip and port
3)	gprs123456  to start gprs
4)	fix060s***n123456 ( TRACKER WILL NOT REPLY FOR THIS COMMAND) for one minute upload

**Comment by manjoel** [comment]
I did this but i still have the same result, every  3 Minutes an acc alsrm

**Comment by kuldiplive** [comment]
Try to Hard Reset. 
From below link ask for direct help they have skype contact. 

http://www.coban.net/En/Contact/Index.html

**Comment by manjoel** [comment]
Ok now i powered the tracker with an lipo 4s 14,8 V, now the acc alarm stoped, and with les gprs it stops sending an message every minute to the server after 5-10 Minutes.

could it be that it's impossible to use the tracker without external power on a server?

**Comment by kuldiplive** [comment]
Not experienced much, but most of the device has internal battery only 
for backup and send power cut alert.

------ Original Message ------

**Comment by nelim** [comment]
yes, connect acc together with power wire, i'm currently using one that way, otherwise it can drain the car battery if not driven too long, that way it will think acc is connected at start :)

**Comment by kuldiplive** [comment]
Don't worry abt car battery,  it will survive long.

------ Original Message ------

**Comment by manjoel** [comment]
When I connect the acc with the power wire together on the battery, the tracker will think acc is on so it stais on like when im driving using full power? Or did you mean usinge an ACC line in the car to power the tracker?

**Comment by kuldiplive** [comment]
It almost uses same power

------ Original Message ------

**Comment by nelim** [comment]
@kuldiplive do you know what is power consumation when on acc and when only powered?
@manjoel i mean to test fully powered, if it does not send gps coordinates, but only ACC alarm(even with acc off), looks like this is device problem and it is not related with traccar server as owner said.

**Comment by kuldiplive** [comment]
Try their Support! 

Hope it solves the issue.

**Comment by kuldiplive** [comment]
Sorry only used coban 103, never with
303.

As now we mostly use Gt06n, and concox devices are working well
